### PR TITLE
Version 1.0

### PR DESCRIPTION
**Pull request by ShawnGilleran**
Adding initial version of the LiveRamp Identity Resolution and Translation native application quick start guide.

### CODE REVIEW

**Comment by sfc-gh-lambrosetti** [comment]
Shawn - this looks great! Because of the amount of code included here. Is it possible that you can provide a notebook of the code in different cells? Otherwise, this quickstart looks great!

File: site/sfguides/src/liveramp_identity_and_translation_quickstart/liveramp_identity_and_translation_quickstart.md [hunk 1]
```diff
@@ -0,0 +1,448 @@
...
+```
+
+After creating make sure to run a quick select on the table to make sure it has what you expect.
+```
+select * from NewTable limit 100;
+```
+
+
+
+## Summary and Next Steps
```
Comment (iamontheinet) [inline site/sfguides/src/liveramp_identity_and_translation_quickstart/liveramp_identity_and_translation_quickstart.md]: @ShawnGilleran @sfc-gh-lambrosetti Rename this to "Conclusion And Resources"

File: site/sfguides/src/liveramp_identity_and_translation_quickstart/liveramp_identity_and_translation_quickstart.md [hunk 2]
```diff
@@ -0,0 +1,448 @@
+id: liveramp_identity_and_translation_quickstart
+summary: Identity Resolution and Transcoding with LiveRamp and Snowflake
+categories: Marketing,Getting-Started,Partner-Integrations
+environments: web
+status: Published 
+feedback link: https://github.com/Snowflake-Labs/sfguides/issues
+tags: Getting Started, Data Science, Data Engineering, MarTech, Marketing
+
+# Identity Resolution and Transcoding with LiveRamp and Snowflake
```
Comment (iamontheinet) [inline site/sfguides/src/liveramp_identity_and_translation_quickstart/liveramp_identity_and_translation_quickstart.md:9]: @ShawnGilleran @sfc-gh-lambrosetti Missing "What You'll Need" section with at least the need to have a Snowflake account?

**Review by iamontheinet — CHANGES_REQUESTED** [review]

File: site/sfguides/src/liveramp_identity_and_translation_quickstart/liveramp_identity_and_translation_quickstart.md (hunk 2 above)
Comment (ShawnGilleran) [reply site/sfguides/src/liveramp_identity_and_translation_quickstart/liveramp_identity_and_translation_quickstart.md:9]: @iamontheinet I have pushed a new commit to resolve the items

**Pull request merged**